--- lua/src/loslib.c.orig
+++ lua/src/loslib.c
@@ -139,6 +139,7 @@ static time_t l_checktime (lua_State *L, int arg) {
 
 
 static int os_execute (lua_State *L) {
+#ifndef IOS_BUILD
   const char *cmd = luaL_optstring(L, 1, NULL);
   int stat = system(cmd);
   if (cmd != NULL)
@@ -147,6 +148,10 @@ static int os_execute (lua_State *L) {
     lua_pushboolean(L, stat);  /* true if there is a shell */
     return 1;
   }
+#else
+  lua_pushboolean(L, 0);
+  return 1;
+#endif
 }
 
 
